code/game/gamemodes/gang/gang.dm

```diff
@@ -387,20 +387,19 @@ GLOBAL_LIST_INIT(gang_outfit_pool, list(/obj/item/clothing/suit/jacket/leather,/
 	return gang_bosses
 
 /datum/game_mode/proc/shuttle_check()
-
 	if(SSshuttle.emergencyNoRecall)
 		return
 	var/alive = 0
 	for(var/mob/living/L in GLOB.player_list)
 		if(L.stat != DEAD)
 			alive++
-	if((alive <= (GLOB.joined_player_list.len * 0.5)) && (SSshuttle.emergency.mode == SHUTTLE_RECALL || SSshuttle.emergency.mode == SHUTTLE_IDLE || (SSshuttle.emergency.timeLeft(1) > (SSshuttle.emergencyCallTime * 0.4))))
+	if((alive <= (GLOB.player_list.len * 0.45)) && (SSshuttle.emergency.mode == SHUTTLE_RECALL || SSshuttle.emergency.mode == SHUTTLE_IDLE || (SSshuttle.emergency.timeLeft(1) > (SSshuttle.emergencyCallTime * 0.4))))
```

Comment:
That's an uncalled shuttle right? The issue before this PR was that it would only trigger if someone had already called the shuttle.